crates/bin/pd/src/migrate/testnet72.rs

```diff
@@ -1,5 +1,5 @@
 //! Contains functions related to the migration script of Testnet72
-
+#![allow(dead_code)]
```

Comment:
Why was this added to the old migrations?

Reply:
They are no longer callable via `pcli` (since they are only valid for the penumbra version contemporary to the migration script) but we wanted to keep the code around to serve as an example of how to write a migration.